infrastructure/main.tf

```
@@ -71,6 +71,12 @@ data "azurerm_key_vault_secret" "idam_service_key" {
   key_vault_id = "${data.azurerm_key_vault.s2s_vault.id}"
 }
 
+resource azurerm_key_vault_secret "idam_service_secret" {
+  name = "microservicekey-ccd-gw"
```

Better rename this (if needed) 
```suggestion
  name = "s2s-microservicekey-ccd-gw"
```